content/en/apps/guides/debugging/SSH-tunnel-for-TLS-termination.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "DIY ngrok/pagekite for Android app testing"
+linkTitle: "DIY ngrok/pagekite for Android app testing"
+weight: 
+description: >
+  SSH Tunnel to a Linux host for TLS termination
+relatedContent: >
+  
+---
+
+## Overview
+To avoid using ngrok or pagekite to allow remote access to your dev instance (https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md#ngrok), you can use a remote linux server to terminate HTTPS connections with free Let's Encrypt certs and reverse proxy this traffic back to a local dev instance over an SSH tunnel:
```

Comment:
For the overview I would rework it in a way that is understandable to a reader who does not know as much about the technical aspects, and focus more on what you are covering rather than what you are not doing eg using ngrok/pagekite.